checkstyle-sonar-plugin/src/main/java/org/sonar/plugins/checkstyle/CheckstyleProfileExporter.java

```diff
@@ -102,6 +102,7 @@ private void generateXml(Writer writer, Map<String,
             List<ActiveRuleWrapper>> activeRulesByConfigKey) throws IOException {
         appendXmlHeader(writer);
         appendCustomFilters(writer);
+        appendTabWidth(writer);
```

Comment:
Please print before filters.

Config is exportable, it is better to print in good way

Reply:
moved it up to get it printed before filters.